packages/vscode-extension/src/commandsTreeViewProvider.ts

```diff
@@ -4,13 +4,14 @@
 import * as vscode from "vscode";
 import * as path from "path";
 import { ext } from "./extensionVariables";
```

Comment:
how about put this ts file also under tree folder?

Reply:
Will do in next pr.